cmd/sdk-server/client_gen.go

```diff
@@ -0,0 +1,595 @@
+}
+
+func (ch *clientHandler) CreateCampaign(ctx *apiserv.Context) apiserv.Response { // method:POST
+	c := ch.getClient(ctx)
+	if ctx.Done() {
+		return nil
+	}
+
+	var cmp *sdk.Campaign
+	if err := ctx.BindJSON(&cmp); err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	data, err := c.CreateCampaign(context.Background(), ctx.Param("uid"), cmp)
+	if err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	return apiserv.NewJSONResponse(data)
+}
+
+func (ch *clientHandler) CreateDraftCampaign(ctx *apiserv.Context) apiserv.Response { // method:POST
+	c := ch.getClient(ctx)
+	if ctx.Done() {
+		return nil
+	}
+
+	var cmp *sdk.Campaign
+	if err := ctx.BindJSON(&cmp); err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	data, err := c.CreateDraftCampaign(context.Background(), ctx.Param("uid"), cmp)
+	if err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	return apiserv.NewJSONResponse(data)
+}
+
+func (ch *clientHandler) CreateFullCampaign(ctx *apiserv.Context) apiserv.Response { // method:POST
+	c := ch.getClient(ctx)
+	if ctx.Done() {
+		return nil
+	}
+
+	var req *sdk.CreateFullCampaignRequest
+	if err := ctx.BindJSON(&req); err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	data, err := c.CreateFullCampaign(context.Background(), ctx.Param("uid"), req)
+	if err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	return apiserv.NewJSONResponse(data)
+}
+
+func (ch *clientHandler) CreateProximitySegment(ctx *apiserv.Context) apiserv.Response { // method:POST
+	c := ch.getClient(ctx)
+	if ctx.Done() {
+		return nil
+	}
+
+	var seg *sdk.ProximitySegment
+	if err := ctx.BindJSON(&seg); err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	data, err := c.CreateProximitySegment(context.Background(), ctx.Param("uid"), seg)
+	if err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	return apiserv.NewJSONResponse(data)
+}
+
+func (ch *clientHandler) CreateSegment(ctx *apiserv.Context) apiserv.Response { // method:POST
+	c := ch.getClient(ctx)
+	if ctx.Done() {
+		return nil
+	}
+
+	var seg *sdk.Segment
+	if err := ctx.BindJSON(&seg); err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	data, err := c.CreateSegment(context.Background(), ctx.Param("uid"), seg)
+	if err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	return apiserv.NewJSONResponse(data)
+}
+
+func (ch *clientHandler) DeleteAd(ctx *apiserv.Context) apiserv.Response {
+	c := ch.getClient(ctx)
+	if ctx.Done() {
+		return nil
+	}
+
+	data, err := "OK", c.DeleteAd(context.Background(), ctx.Param("adID"))
+	if err != nil {
+		return apiserv.NewJSONErrorResponse(http.StatusBadRequest, err)
+	}
+
+	return apiserv.NewJSONResponse(data)
```

Comment:
https://github.com/missionMeteora/apiserv/commit/e1ee8286613ac52f42989eef5d77f2ae55b5943b#diff-79e51482e93fbb1dae034c6f60053860R20 Can't u just use this

Reply:
Same goes for all the other "OK"s I see